src/index.js

```diff
@@ -403,6 +614,39 @@ export default function ({ types: t }) {
         fixReactNativeViewAttributes(path);
         fixTouchableMixin(t, path);
       },
+      JSXAttribute(path, { opts }) {
+        if (opts.isNewArchitectureEnabled) {
+          extendExistingRef(path);
+        }
+
+        // disable view optimization for only View component
+        if (path.parent.name.name !== 'View') return;
```

Comment:
Should this be checked against the `elements` [list](https://github.com/fullstorydev/fullstory-babel-plugin-react-native/pull/29/files#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R5-R18)?

Reply:
View optimization only is relevant to `<View />` element, so any other elements are not involved.